homeassistant/components/switch/flux.py

```diff
@@ -136,7 +136,7 @@ def turn_on(self, **kwargs):
         """Turn on flux."""
         self._state = True
         self.unsub_tracker = track_utc_time_change(self.hass, self.flux_update,
```

Comment:
Instead of passing `local=True`, use `track_time_change`. 

Changing a date to be local instead of UTC probably has more side effects.

For example there is logging that wraps the `now` value with `as_local`